docs/src/main/asciidoc/datasource.adoc

```diff
@@ -296,16 +296,15 @@ Because of this, you can also use `javax.sql.DataSource` as the injected type.
 
 ===== Oracle considerations
 
-As documented in https://github.com/quarkusio/quarkus/issues/36265[issue #36265],
-Oracle has a very weird behavior of committing the uncommitted transactions on connection closing.
+As documented in link:https://github.com/quarkusio/quarkus/issues/36265[issue #36265],
+Oracle unexpectedly commits uncommitted transactions when closing a connection.
+This means that when stopping Quarkus, in-progress transactions might be committed even if they are incomplete.
 
-Which means that when stopping Quarkus for instance, in progress transactions might be committed even if incomplete.
+Since this behavior is unexpected and could lead to data loss, an interceptor rolls back any unfinished transactions when closing a connection, unless XA transactions are used, in which case the transaction manager handles the rollback.
```

Comment:
```suggestion
Because this behavior is unexpected and can lead to data loss, an interceptor rolls back any unfinished transactions when closing a connection. 
However, if you use XA transactions, the transaction manager handles the rollback.
```